symplyphysics/laws/kinematic/impuls.py

```diff
@@ -0,0 +1,30 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+                           validate_input, validate_output)
+
+
+# Descroption
+## The momentum of a body (the amount of motion) is a vector physical quantity, the modulus of which is equal to the product of the mass of the body by its speed.
+## p = m * v
```

Comment:
We already have this: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/definitions/momentum_is_mass_times_velocity.py

Reply:
I'm sorry.